restapi/src/main/java/io/stargate/web/impl/RestApiServer.java

```diff
@@ -131,13 +129,26 @@ protected void configure() {
                 bind(db).to(Db.class);
               }
             });
+    final RestDBAccessFactory restDBFactory =
+        new RestDBAccessFactory(authenticationService, authorizationService, dataStoreFactory);
+    environment
+        .jersey()
+        .register(
+            new AbstractBinder() {
+              @Override
+              protected void configure() {
+                bind(restDBFactory).to(RestDBAccessFactory.class);
```

Comment:
is there any chance we can group the stuff in one `AbstractBinder` instead of creating a new one for each registration here.. It's just easier to read at the end? We also extracted docs stuff into an own binder..

Reply:
I was following an existing pattern, no reason for not combining. Will change this.

Reply:
Combined -- not sure what'd be the cleanest way wrt grouping (REST vs Doc API), but at least less verbose now.